src/Language/PureScript/Interactive/Message.hs

```diff
@@ -28,27 +32,27 @@ helpMessage = "The following commands are available:\n\n    " ++
 
   extraHelp =
     "Further information is available on the PureScript documentation repository:\n" ++
-    " --> https://github.com/purescript/documentation/blob/master/PSCi.md"
+    " --> " ++ guideURL
 
 -- | The welcome prologue.
 prologueMessage :: String
 prologueMessage = unlines
-  [ "PSCi, version " ++ showVersion Paths.version
+  [ "PureScript REPL, version " ++ showVersion Paths.version
```

Comment:
I think it's fine to continue to call the app "PSCi", but when referring to the command, it should say `purs repl`.